examples/network-manager-app/linux/args.gni

```diff
@@ -22,3 +22,4 @@ chip_project_config_include_dirs = [
 ]
 
 chip_config_network_layer_ble = false
+chip_enable_access_restrictions = true
```

Comment:
Please indicate with a comment that this is what enables the ARL support and the NIM sample app requires it.

Reply:
done